ontology/lipro.md

```diff
@@ -7,7 +7,7 @@ build:
   source_url: http://www.lipidprofiles.com/LipidOntology
 contact:
   email: bakerc@unb.ca
-  label: Christipher Baker
```

Comment:
@egonw do you know the ORCID and github handles for this person?

Reply:
Should be him:

* https://orcid.org/0000-0003-4004-6479
* https://scholia.toolforge.org/author/Q59389381

Reply:
can you add this also under `label:` as `orcid: 0000-0003-4004-6479` then we can accept this PR!